src/components/QuotesContainer.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from "react";
+import QuotesText from "./QuotesText";
+import Image from "next/image";
+
+export default function QuotesContainer() {
+  const [quotesText, setQuotesText] = useState(Quotes[0].quote);
+  const [buttonHighlighted, setButtonHighLighted] = useState(0);
+
+  return (
+    <div className="bg-background-quotes flex justify-center p-20">
+      <div className="flex items-center justify-center flex-col text-gray-200 font-serif">
+        <h1 className="text-2xl flex py-6">Don't just take our word for it</h1>
+        <QuotesText
+          Quotes={Quotes}
```

Comment:
seems odd that this is caplitalized, seems like an anti pattern

Reply:
addressed for the entire PR.